src/entity/Comment.ts

```diff
@@ -70,9 +70,13 @@ export class Comment {
   })
   parent: Promise<Comment>;
 
+  mentions?: string[];
```

Comment:
We agreed on implementing it as another table rather than arrays as far as I remember.
This way it can be easier to track when someone changes their username.
What do you think?

Reply:
Yes. This property is non-existent in the db. It is just a placeholder for when the comment is created or updated, the handler for generating the markdown has a reference to those should be mentioned.

Reply:
I think this can lead to confusion when using this entity. Some developers might forget to set this property before saving the object and then it can be invalid. We don't have any failsafe mechanism to make sure other devs use this property correctly

Reply:
That makes total sense. In that case, we will have to take out the function inside the comment entity from instead automatically trigger generating the markdown to manually do it. With that, I will create a generic function to utilize in both cases (insert, update).